torch/onnx/_internal/exporter.py

```diff
@@ -687,9 +687,7 @@ def __init__(
     def __call__(
         self,
         *args: Any,
-        model_with_state_dict: Optional[
-            Union[torch.nn.Module, Callable, torch_export.ExportedProgram]
-        ] = None,
+        exported_program: Optional[torch_export.ExportedProgram] = None,
```

Comment:
we are not deprecating the non torchexported program just yet. we should keep the torch.nn.module type hint

Reply:
Reverting it back. But wanted to remind, if users pass in nn.Module, it doesn't make any effect.